modules/oxide-components/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@tiny/oxide-components",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinymce/tinymce.git",
+    "directory": "modules/oxide-components"
+  },
+  "description": "React component library for TinyMCE rich text editor",
+  "author": "Ephox Corporation DBA Tiny Technologies, Inc",
+  "bugs": {
+    "url": "https://github.com/tinymce/tinymce/issues"
+  },
+  "license": "GPL-2.0-or-later",
+  "sideEffects": false,
+  "files": [
+    "dist",
+    "src",
+    "tsconfig.json",
+    "README.md",
+    "LEGAL.txt",
+    "CHANGELOG.txt",
+    "LICENSE.txt"
+  ],
+  "main": "dist/main.js",
+  "module": "dist/main.js",
+  "types": "dist/main.d.ts",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint --max-warnings=0 src/**/*.tsx",
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "test-watch": "vitest",
+    "test": "echo 'No tests'",
+    "test-manual": "playwright install && playwright install-deps && vitest --watch=false --reporter=junit --outputFile=test-results.xml",
```

Comment:
Please use a scratch folder or something else that will be git ignored when used locally.

And why is it called `test-manual` when it's used for what seems to be mostly automated CI purposes?

Reply:
the `test-resluts.xml` is ignored by git. Does it have to be in the scratch folder or does it suffice to be ignored by git ?

I just changed `test-manual` to `test-ci`

In this library I think we need vitest as it has good integration with storybook for testing our react components

Reply:
I'd still prefer to use a scratch folder for generated reports and logs to match how our other projects operate.

Reply:
Done!